jenkins/databricks/clusterutils.py

```diff
@@ -23,21 +23,23 @@ class ClusterUtils(object):
 
     @staticmethod
     def generate_create_templ(sshKey, cluster_name, runtime, idle_timeout,
-            num_workers, driver_node_type, worker_node_type,
+            num_workers, driver_node_type, worker_node_type, cluster_type,
```

Comment:
Add 'cluster_type' parameter to make create.py common for aws and azure Databricks

Reply:
don't we need whatever equivalent for azure or those just aren't specified?

Reply:
also perhaps a better name for this would be like cloud_provider.

Reply:
> don't we need whatever equivalent for azure or those just aren't specified?

I thinks we need not specify `whatever equivalent for azure`, as the default(unspecified) value of `cluster_type` is aws. 

Also `cluster_type` is for aws configs of creating cluster as below, any other values means we do not need the below configs. So we only need to specify the default(unspecified) value `cluster_type` as 'aws' in the create.py.

        if (cluster_type == 'aws'):
            templ['aws_attributes'] = {
                        "zone_id": "us-west-2a",
                        "first_on_demand": 1,
                        "availability": "SPOT_WITH_FALLBACK",
                        "spot_bid_price_percent": 100,
                        "ebs_volume_count": 0
            }

Reply:
> also perhaps a better name for this would be like cloud_provider.

Sounds good, let me change it.